app/models/ticket.rb

```diff
@@ -1,11 +1,17 @@
 # frozen_string_literal: true
 
-class Ticket < ActiveModelSerializers::Model
-  attributes :user_id, :ip, :created_at
+class Ticket
+  include ActiveModel::Model
+
+  attr_accessor :user_id, :ip, :created_at
 
   validates :user_id, :ip, :created_at, presence: true
   validate :created_at_must_be_less_than_5_minutes
 
+  def id
+    0
```

Comment:
whats this for?

Reply:
It's because jsonapi.rb enforces the resources to return an ID, and given this ticket object is not in the database I just decided to always pass a 0 as the ID.